kafka-zookeeper/install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -eux
+
+echo "*** Installing Kafka and dependencies"
+apk add --update --no-cache jq curl
+
+APACHE_MIRROR=$(curl --stderr /dev/null https://www.apache.org/dyn/closer.cgi\?as_json\=1 | jq -r '.preferred')
+
+# download kafka binaries
+curl -sSL $APACHE_MIRROR/kafka/$KAFKA_VERSION/kafka_$SCALA_VERSION-$KAFKA_VERSION.tgz | tar xz
+mv kafka_$SCALA_VERSION-$KAFKA_VERSION/* .
+
+# Remove bash as our images doesn't have it, and it isn't required
+sed -i 's~#!/bin/bash~#!/bin/sh~g' /kafka/bin/*sh
+
+# Set explicit, basic configuration
+cat > config/server.properties <<-EOF
+broker.id=0
+zookeeper.connect=127.0.0.1:2181
+replica.socket.timeout.ms=1500
+log.dirs=/kafka/logs
+auto.create.topics.enable=true
+offsets.topic.replication.factor=1
+listeners=PLAINTEXT://0.0.0.0:9092,PLAINTEXT_HOST://0.0.0.0:19092
+listener.security.protocol.map=PLAINTEXT:PLAINTEXT,PLAINTEXT_HOST:PLAINTEXT
```

Comment:
add a comment that advertised listeners will be appended later.

Reply:
done.